### ISSUE TITLE
[Feat]: Add a new theme to Super Timer

### ISSUE THREAD

**Issue by samyakshah3008** [samyakshah3008/pomo-super-focus-webapp#86]
## Description: 

Currently we only have one theme - TMKOC, but to consider all audience who are international or maybe non tmkoc fan or a girl who wants to have a Girly theme, we need more themes. 

Note: 
Currently on research which next new theme to be added - ideas are most welcomed in comments!

**Comment by CHIRANTH-24** [comment]
Are we planning to change only the placeholders at the moment?

**Comment by mnmnk43434** [comment]
Hello, we tried to solve the issue.

This is what we did:

Implement a new 'Girly' theme option by modifying the theme.ts file to include new theme assets, updating the settings UI in section-four.tsx to allow theme selection, and adjusting the constants.ts file to include the new theme color.

You can review changes in this commit: https://github.com/mnmnk43434/samyakshah3008-pomo-super-focus-webapp-86/commit/d28125e86e2929b2e53d8b171b2d68dbe9572074.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by samyakshah3008** [comment]
> Are we planning to change only the placeholders at the moment?

Yes - instead of tmkoc theme, we will add a new theme as a second option. 
You can check theme.ts file and would love to know which theme you want to implement!

**Comment by CHIRANTH-24** [comment]
Im planning to do it for BTS!

**Comment by bhaveshxrawat** [comment]
A theme for Clash of Clans maybe, where either Barbarian King and Archer Queen have their weapon on you if you get distracted? Curious, if I could get asset for it.

**Comment by CHIRANTH-24** [comment]
Awesome @bhaveshxrawat

**Comment by samyakshah3008** [comment]
Haha that's nice theme - on my mind. But it will be quite difficult to get assets for the same, need to research. Thanks tho @bhaveshxrawat if you find something, please let me know or feel free to directly send a PR with that theme, or even if you send me the sources of assets, that will be counted as contribution!
